features/admin.actions.v1/util/form-field-util.ts

```diff
@@ -109,3 +113,93 @@ export const validateActionCommonFields = (
 
     return errors;
 };
+
+/**
+ * Validate endpoint fields of the ActionConfig form.
+ *
+ * @param values - Form values to validate.
+ * @param options - Dynamic options for validation (e.g., flags and authenticationType).
+ * @param t - localization translations
+ * @returns A partial object containing validation errors.
+ */
+export const validateActionEndpointFields = (
+    values: EndpointConfigFormPropertyInterface,
+    options: {
+        isCreateFormState: boolean;
+        isAuthenticationUpdateFormState: boolean;
+        authenticationType?: AuthenticationType;
+    },
+    t: (key: string, options?: any) => string
```

Comment:
We don't need to pass this as a param. Use `I18n` instance from core module instead.

eg: `I18n.instance.t("actions:<your_i18n_key>")`